docs/source/contents/getting-started/kubernetes-installation/security/strimzi-sasl.md

```diff
@@ -18,6 +18,8 @@ This will call the Strimzi cluster Helm chart provided by the project with overr
 Install Core v2 with SASL settings using a custom values file.
 This sets the secret created by Strimzi for the user created above (`seldon`) and targets the server certificate authority secret from the name of the cluster created on install of the Kafka cluster (`seldon-cluster-ca-cert`).
 
+Configure Seldon Core v2 by setting following Helm values:
+
 ```{literalinclude} ../../../../../../k8s/samples/values-strimzi-kafka-sasl-scram.yaml
```

Comment:
:thought_balloon: :wrench: The file referenced here does not explicitly set the SASL mechanism, thus is reliant on the default, which may change.  Ideally, these examples should be fully self-sufficient

Reply:
Fixed in follow-up PR: https://github.com/SeldonIO/seldon-core/pull/5130/commits/708bc676f196ecf540a94dacbc5ebddf75543365